pythonosc/test/test_osc_message_builder.py

```diff
@@ -20,6 +20,10 @@ def test_wrong_param_raise(self):
     builder = osc_message_builder.OscMessageBuilder("")
     self.assertRaises(ValueError, builder.add_arg, "what?", 1)
 
+  def test_add_arg_invalid_infered_type(self):
+    builder = osc_message_builder.OscMessageBuilder('')
```

Comment:
oh well... I just noticed this file is a mess in terms of consistency.. please ignore I'll fix all of that later on